analyzers/python/python.go

```diff
@@ -163,15 +163,29 @@ func (a *Analyzer) Analyze() (graph.Deps, error) {
 	case "requirements":
 		fallthrough
 	default:
-		reqs, err := pip.FromFile(a.requirementsFile(a.Module))
-		if err != nil {
-			return graph.Deps{}, err
+		requirementsPath := a.requirementsFile(a.Module)
+		reqs, err := pip.FromFile(requirementsPath)
+		if err == nil {
+			return requirementsToDeps(reqs), nil
 		}
-		imports := FromRequirements(reqs)
-		return graph.Deps{
-			Direct:     imports,
-			Transitive: fromImports(imports),
-		}, nil
+
+		setupPyPath := filepath.Join(a.Module.Dir, "setup.py")
+		reqs, setupErr := pip.FromSetupPy(setupPyPath)
+		if setupErr == nil {
+			return requirementsToDeps(reqs), nil
+		}
+
+		err.Troubleshooting = err.Troubleshooting + " OR " + setupErr.Troubleshooting
```

Comment:
We might want to add something to the errors package for combining two errors for fallback situations like these. It could, for example, combine the `Troubleshooting` with "OR", cause / message with "AND", ...